Automattic/wp-calypso
# Add list of individual products with tooltips [Offer jetpack complete project]

Related to #
  - 0-as-1203791007934513
  - 0-as-1203791007934524
  - 0-as-1203791007934518

## Proposed Changes
As a part of 'Offer jetpack Complete after plugin activation' project this PR adds included single product component with a specific tooltip and organises single products into a table adjustable for different screens.

Appropriate Tracking events are also added.

## Testing Instructions



- Create a new JN site
- Spin up this PR by running `yarn start` on a branch OR simply use Calypso Live link provided below.
- On the pinned environment navigate to `/jetpack/connect/plans/complete`
- You should see the list of individual products according to the designs. And they should behave correctly on different screen sizes.
- You could check that tooltip opens correctly for each item and navigates properly after clicking `Learn More...`
- You could also check using Redux toolkit that tracking events are fired.


## Pre-merge Checklist
